webapp/src/blocks/card.ts

```diff
@@ -12,6 +12,7 @@ type CardFields = {
 
 type Card = Block & {
     fields: CardFields
+    limited: boolean
```

Comment:
question: could this just be added to the Block type? similar to in my PR: https://github.com/mattermost/focalboard/blob/a863e43fb0876d21d7996d92f5c3452a99c25437/webapp/src/blocks/block.ts#L44

Reply:
Yes, nice catch. I will update it.